config/config_test.go

```diff
@@ -190,23 +192,23 @@ func TestGetOptionalParameters(t *testing.T) {
 	t.Run("Live data without sensors", func(t *testing.T) {
 		cfgService := makeCfgService()
 		cfgService.Attributes["use_live_data"] = true
-		cfg, err := newAttrConfig(cfgService)
+		cfg, err := newConfig(cfgService)
 		test.That(t, err, test.ShouldBeNil)
 		_, _, _, _, _, err = GetOptionalParameters(cfg, "localhost", 1001, 1002, logger)
 		test.That(t, err, test.ShouldBeError, newError("sensors field cannot be empty when use_live_data is set to true"))
 	})
 }
 
-func newAttrConfig(cfg config.Service) (*AttrConfig, error) {
-	attrCfg := &AttrConfig{}
-
-	if _, err := config.TransformAttributeMapToStruct(attrCfg, cfg.Attributes); err != nil {
-		return &AttrConfig{}, newError(err.Error())
+func newConfig(conf resource.Config) (*Config, error) {
+	slamConf, err := resource.TransformAttributeMap[*Config](conf.Attributes)
+	if err != nil {
+		return &Config{}, newError(err.Error())
 	}
 
-	if _, err := attrCfg.Validate("services.slam.attributes.fake"); err != nil {
-		return &AttrConfig{}, newError(err.Error())
+	// This temporary value will be replaced once we are using rdk's validation
```

Comment:
Could you explain more what this means? Is there a ticket to change this?

Reply:
I'll remove this. It was a copy & paste

Reply:
ah so it's not relevant here?

Reply:
Correct

Reply:
ty!